Server/tmp/upload/55b62868cf4cf59724816bf19843575e

```diff
@@ -0,0 +1,242 @@
+{
+  "reference": {
```

Comment:
We do not need these files. They created if you test an upload. Avoid committing these to the repo please

Reply:
tmp folder has been deleted/.